src/pages/Accounts.vue

```diff
@@ -22,47 +13,7 @@
         active
       />
 
-      <template v-if="inactiveIdentities.length === 0">
-        <p>
-          This is your first account, it enables you to use our æpps,
-          get Tokens, trade them and much more!
-        </p>
-        <p>
-          Quickly activate another account or instantly create one or multiple accounts.
-          Each has it’s own address and Token Balance
-        </p>
-      </template>
-      <template v-else>
-        <ae-divider/>
-        <label>
-          Inactive
-          <span>{{ inactiveIdentities.length }}</span>
-        </label>
-        <div class="inactive-accounts">
-          <ae-identity
-            v-for="{ identity, index, beforeActive, active } in inactiveIdentities"
-            :key="identity.address"
-            :identity="identity"
-            :class="{ 'before-active': beforeActive, active }"
-            collapsed
-            @click="activateCard(index)"
-          >
-            <div
-              v-if="active"
-              class="action-buttons">
-              <ae-divider />
-              <ae-button
-                size="small"
-                type="dramatic"
-                uppercase
-                @click="selectIdentity(index)"
-              >
-                Activate
-              </ae-button>
-            </div>
-          </ae-identity>
```

Comment:
Was it reasonable to extract actually the same accounts rendering cycle to IdentityRadioList component?

Reply:
I do not quite understand. Do you suggest that all of AccountSwitcher component should be in Accounts.vue component or that the AccountSwitcher should send signal to Accounts.vue and the method to change account be called from there?

Reply:
I mean the first one, I have made it.